app/models/datawarehouse_node.rb

```diff
@@ -0,0 +1,9 @@
+class DatawarehouseNode < ApplicationRecord
```

Comment:
@moolitayer Someone might want to make an SQL / MongoDB / something else Datawarehouse Manager and might want to use those nodes..